game-core/src/main/java/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -58,8 +55,14 @@
 
   /** Identifies the location where a nation flag is drawn relative to a unit. */
   public enum UnitFlagDrawMode {
-    LARGE_FLAG,
-    SMALL_FLAG
+    NONE,
+    SMALL_FLAG,
+    LARGE_FLAG;
+
+    public UnitFlagDrawMode nextDrawMode() {
+      final var values = UnitsDrawer.UnitFlagDrawMode.values();
```

Comment:
Is it really necessary to put the full enum name here?
If yes I'd actually be surprised because I'd consider values() a static method of every enum

Reply:
No, copy/paste artifact. Any static method call can be fully qualified, AFAIK as an unwritten convent, we conventionally do not do this. Updated: adf776ea8